portfolio/src/main/webapp/index.html

```diff
@@ -25,10 +25,9 @@ <h3 id="header"><i>About me</i> &#128515;:</h3>
       <hr>
       <h3 id="header"><i>Fun facts</i> &#128175;:</h3>      
       <p>Click here to get a random <strong>fact</strong> about me: 
-        <button style="background-color: rgba(105, 105, 105, 0.205);" onclick="addRandomFact()">&#128272;</button>
+        <button style="background-color: rgba(105, 105, 105, 0.205);" onclick="getRandomFact();">&#128272;</button>
       </p>
-      <div id="greeting-container"></div>
-      <p> </p>
+      <div id="fact-container"></div>
```

Comment:
Looks like you're just using this div to add spacing by adding padding at the bottom. As opposed to creating an empty div (which is confusing because a div is just a container for other stuff), I would just add the padding to the `<p>` above it by giving the paragraph a class.